LoopKitUI/View Controllers/SettingsNavigationViewController.swift

```diff
@@ -17,3 +17,37 @@ open class SettingsNavigationViewController: UINavigationController, CompletionN
     }
 
 }
+
+open class CGMManagerSettingsNavigationViewController: SettingsNavigationViewController, CGMManagerOnboardNotifying, PreferredGlucoseUnitObserver {
```

Comment:
nit: I think this class falls into that class of code we discussed yesterday that happens to be shared behavior, but might not make sense to be prescriptive. And I'd have no problem if this was kept and copied in individual CGM/Pump managers. No need to change this; I think it's ok as is. But that's the reaction I have when I see it.

Reply:
I agree that it could go either way. If we were to push this all into the CGM/Pump managers then I'd recommended doing away with `SettingsNavigationViewController` entirely using the same logic. On the other hand, this is just a basic implementation and it doesn't force the manager from using it. They can implement their own.

Which way would you like me to go? Push everything into CGM/Pump manager or keep as-is?

Reply:
I'm fine with it here, but it leaves some aji. :)